File: hw/vizcaino/u3/hw28Strategy/hw28StrategyJavaScript/sortApp.js
const SortingContext = require('./sortingContext');

class SortApp {
    static main() {
        const data = [3, 6, 4, 6, 7, 8, 5, 6, 7, 5, 3, 3];
        const sc = new SortingContext();
        const sortedList = sc.sort(data);

        console.log("Sorted data:", sortedList.join(" "));
    }
}

SortApp.main();

File: hw/vizcaino/u3/hw28Strategy/hw28StrategyPython/sort_app.py
from sorting_context import SortingContext

class SortApp:
    @staticmethod
    def main():
        data = [6]
        sc = SortingContext()
        sorted_list = sc.sort(data)

        print("Sorted data:", " ".join(map(str, sorted_list)))

if __name__ == "__main__":
    SortApp.main()